packages/shared/src/components/profile/SourceList.tsx

```diff
@@ -55,85 +55,89 @@ export const SourceList = ({
         placeholder={loader}
       >
         {sources.map((source) => (
-          <Link
-            key={source.id}
-            href={`${webappUrl}${
-              source.type === SourceType.Squad ? 'squads' : 'sources'
-            }/${source.handle}`}
-          >
-            <a
+          <div className="flex gap-2 px-6 py-3 hover:bg-surface-hover">
+            <Link
               key={source.id}
-              className="flex gap-2 px-6 py-3 hover:bg-surface-hover"
-              rel={anchorDefaultRel}
-              target="_blank"
+              href={`${webappUrl}${
+                source.type === SourceType.Squad ? 'squads' : 'sources'
+              }/${source.handle}`}
             >
```

Comment:
Only added a wrapper div with the flex classes and moved the link inside using `absolute inset-0` for positioning.